test/validator/strict_dependencies_test.dart

```diff
@@ -17,30 +17,88 @@ Validator strictDeps(Entrypoint entrypoint) {
 }
 
 main() {
-  group('should consider a package valid if', () {
+  group('should consider a package valid if it', () {
     setUp(d.validPackage.create);
 
     integration('looks normal', () => expectNoValidationError(strictDeps));
 
     integration('declares an "import" as a dependency', () {
```

Comment:
We don't need to duplicate every test, but we should probably have at least one test for an `export` that *doesn't* produce a warning.